future.go

```diff
@@ -153,14 +150,8 @@ type restoreFuture struct {
 	ID string
 }
 
-// verifyFuture is used to verify the current node is still
```

Comment:
That was an accident. I think I had deleted the struct entirely and then added it back but forgot the comment.